ehitchhiking-ui/src/app/shared/components/rate-passangers-modal/rate-passangers-modal.component.sass

```diff
@@ -0,0 +1,14 @@
+.table-content
+  .table
+    width: 100%
+  .title
+    text-align: center
+    .table-tr
+      text-align: center
+  button
```

Comment:
It is a bad practice to override styles for selectors. Could you please add custom css classes and override them?

Reply:
fixed